ci_js/doc/index.rst

```diff
@@ -0,0 +1,68 @@
+    * **Skip enabled** - поставить галочку, если нужно добавить кнопку Skip в подсказках
+    * **Using assets** - выбрать нужные ассетсы (assets_frontend, assets_backend, pos_assets)
+* Во вкладке ``Sets`` нажимаем на ``[Add an item]``
+* В появившемся окне в поле **Template** нажимаем на выпадающее меню и выбираем подходящий шаблон.
+* Если нужного шаблона нет, то нажимаем на ``[Create and Edit]``:
+    * В поле **Template name** вводим название шаблона
+    * Во вкладке ``Steps`` нажать ``[Add an item]`` и в появившемся окне заполнить следующие поля:
+        * **content** - будет выводится в tips
+        * **trigger** - селектор элемента, по которому нужно кликнуть в тесте, либо напротив чего разместить указатель(обязательное поле)
+        * **extra_trigger** - клик на ``trigger`` сработает, когда указанный здесь элемент будет видимым.
+        * **timeout** - максимальное время ожидания условия
+        * **position** - позиция для указателя на выбор: left, rigth, top, bottom
+        * **width** - ширина указателя в пикселях. По умолчанию 270px
+        * **edition** - указание того, в каком издании Odoo можно выполнять шаг (community, enterprise or any edition). По умолчанию стоит параметр any edition
+        * **run** - действия которые выполняются, когда тур запущен автоматически
+        * **auto** - шаг будет пропущен в не автоматическом запуске
+    * Если нужно добавить ещё шаг, то нажать ``[Save & New]``, либо ``[Save & Close]``, если шаги закончились.
+    * Нажать ``[Save]``, чтобы сохранить получившийся шаблон
+* Когда шаблон выбран, то под полем ``[Template]`` появляются поля с названиями переменных, которые принимает шаблон(если он их принимает), нужно заполнить эти поля, в соответствии с контекстом
+* Когда все необходимые поля заполнены, можно нажать ``[Save & New]``, если нужно добавить еще один set. Если составление сетов для тура закончено можно нажать ``[Save & Close]``
+* Чтобы сохранить получившийся тур нужно нажать ``[Save]``, после этого увидим наш тур во вкладке ``Tours``
+
+Отладка и быстрый запуск тура
+-----------------------------
+
+* Открыть ``[[ CI JS ]] >> CI JS >> [Tours]``
+* В списке туров найти необходимый и открыть его
+* Нажать на ``[Start tour]``, после чего тур запустится
+
+
+Генерация файла тура
+--------------------
+
+* Открыть ``[[ CI JS ]] >> CI JS >> [Tours]``
+* В списке туров найти необходимый и открыть его
+* Нажать на ``[Download tour]``
+
+Генерация XML-файла для экспорта шаблонов
+-----------------------------------------
+* Открыть ``[[ CI JS ]] >> CI JS >> [Templates]``
+* Нажать ``[Make XML]``, после чего начнется загрузка XML-файла
+
+Генерация JS-кода отдельного сета в туре для отладки
+---------------------------------------------
+* Открыть ``[[ CI JS ]] >> CI JS >> [Tours]``
+* Открыть необходимый тур
+* Найти интересующий сет и нажать ``[Generate JS]``, после чего код можно скопировать из формы.
```

Comment:
@trojikman Please describe in more detail where the button will be located and where the generated code will appear.